scripts/game_structure/load_cat.py

```diff
@@ -215,14 +231,36 @@ def json_load():
             new_cat.previous_mates = (
                 cat["previous_mates"] if "previous_mates" in cat else []
             )
-            new_cat.dead = cat["dead"]
+
+            # checking for old dead
+            if cat.get("dead"):
```

Comment:
```suggestion
            if cat.get("dead") or cat.get("df"): 
```
Combine the block below with this one so they're both handled at the same time